src/main/kotlin/com/beust/kobalt/app/LanguageTemplateGenerator.kt

```diff
@@ -20,6 +20,8 @@ abstract class LanguageTemplateGenerator : ITemplate {
 
     abstract val defaultSourceDirectories : HashSet<String>
     abstract val defaultTestDirectories : HashSet<String>
+    abstract val mainDependencies : ArrayList<Pom.Dependency>
```

Comment:
Could possibly make the case that these should not be abstract & should be initialized with  ArrayList<Pom.Dependency>(), so inheriting generators don't have to do that if they don't need to introduce any dependencies.